### ISSUE TITLE
Missing output parts when cat'ing UTF-8 stress test

### ISSUE THREAD

**Issue by egmontkob** [microsoft/terminal#3147]
# Environment

```none
Windows build number: Win32NT 10.0.18362.0
Windows Terminal version (if applicable): 0.5.2762.0
```

# Steps to reproduce

`ssh` to a Linux box.

Get Markus Kuhn's [UTF-8 decoder capability and stress test](https://www.cl.cam.ac.uk/~mgk25/ucs/examples/UTF-8-test.txt) file.

`cat UTF-8-test.txt`

# Expected behavior

Consistent output (which, by the way, reasonably closely matches what's described in the file), regardless of timing, including with various parameters to [slowcat](https://gitlab.gnome.org/GNOME/vte/blob/0.58.1/src/slowcat.c).

# Actual behavior

Faulty and inconsistent output, presumably depending on the timing of events.

Often the output terminates way too prematurely, e.g. during the intro or section 1 or 2 of the file. Sometimes the succeeding shell prompt isn't printed either, although the shell is already ready to accept a new command.

Other times the output ends as the end of the text file, but as you scroll back you'll notice that a big chunk is missing.

**Comment by skyline75489** [comment]
I've tried it with the original PowerShell embedded in Win10, PowerShell Core Preview 7 and cmd.exe, the problem exists in all the above environment. This isn't really a WT specific issue.

**Comment by miniksa** [comment]
#3380 might fix this, but I can't fully repro the original problem so I'm not certain.

**Comment by egmontkob** [comment]
Indeed fixed in v0.7, thanks!
